modules/virt-about-datavolume-gc.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-deleting-datavolumes.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-datavolume-gc_{context}"]
+
+= About data volume garbage collection
+
+Before {VirtProductName} release 4.12, you used an `oc` command to remove a data volume. This command removes the connection between the data volume and the persistent volume claim (PVC). However, using this command does not always provide the expected result. For example, Kubevirt might attempt to recreate a data volume that you removed if Kubevirt had initially created the volume.
+
+For {VirtProductName} release 4.12 and later, an automated garbage collection process removes data volumes that are no longer associated with the PVC. This eliminates manual clean-up of data volumes.
```

Comment:
There never has been a need to manually clean up Data Volumes.  They simply shared the same lifetime as their PVC.

Reply:
I wondered about this to! Assumed there was a need as we were documenting the manual deletion.

This begs the question: Should we eliminate the list and delete this procedure from older versions of the docs? If so, I can create another story, in addition to the story where we revise the wording to more "virtual machine disk"-speak.